Package/gulpfile.js

```diff
@@ -93,6 +94,89 @@ const copyAndroidFiles = async () => {
+  shelljs.mkdir('-p', 'Assembled/windows/libs/arm/Release');
+  shelljs.mkdir('-p', 'Assembled/windows/libs/arm64');
+  shelljs.mkdir('-p', 'Assembled/windows/libs/arm64/Debug');
+  shelljs.mkdir('-p', 'Assembled/windows/libs/arm64/Release');
+  shelljs.mkdir('-p', 'Assembled/windows/libs/x86');
+  shelljs.mkdir('-p', 'Assembled/windows/libs/x86/Debug');
+  shelljs.mkdir('-p', 'Assembled/windows/libs/x86/Release');
+  shelljs.mkdir('-p', 'Assembled/windows/libs/x64');
+  shelljs.mkdir('-p', 'Assembled/windows/libs/x64/Debug');
+  shelljs.mkdir('-p', 'Assembled/windows/libs/x64/Release');
+  shelljs.mkdir('-p', 'Assembled/windows/BabylonReactNative');
+}
+
+const copyx86DebugUWPFiles = () => {
+  return gulp.src('../Modules/@babylonjs/react-native/submodules/BabylonNative/Build_uwp_x86/**/Debug/**/*.{lib,pri}')
+    .pipe(rename({ dirname: '' }))
+    .pipe(gulp.dest('Assembled/windows/libs/x86/Debug'));
+}
+
+const copyx86ReleaseUWPFiles = () => {
+  return gulp.src('../Modules/@babylonjs/react-native/submodules/BabylonNative/Build_uwp_x86/**/Release/**/*.{lib,pri}')
+    .pipe(rename({ dirname: '' }))
+    .pipe(gulp.dest('Assembled/windows/libs/x86/Release'));
+}
+
+const copyx64DebugUWPFiles = () => {
+  return gulp.src('../Modules/@babylonjs/react-native/submodules/BabylonNative/Build_uwp_x64/**/Debug/**/*.{lib,pri}')
+    .pipe(rename({ dirname: '' }))
+    .pipe(gulp.dest('Assembled/windows/libs/x64/Debug'));
+}
+
+const copyx64ReleaseUWPFiles = () => {
+  return gulp.src('../Modules/@babylonjs/react-native/submodules/BabylonNative/Build_uwp_x64/**/Release/**/*.{lib,pri}')
+    .pipe(rename({ dirname: '' }))
+    .pipe(gulp.dest('Assembled/windows/libs/x64/Release'));
+}
+
+const copyARMDebugUWPFiles = () => {
+  return gulp.src('../Modules/@babylonjs/react-native/submodules/BabylonNative/Build_uwp_arm/**/Debug/**/*.{lib,pri}')
+    .pipe(rename({ dirname: '' }))
+    .pipe(gulp.dest('Assembled/windows/libs/arm/Debug'));
+}
+
+const copyARMReleaseUWPFiles = () => {
+  return gulp.src('../Modules/@babylonjs/react-native/submodules/BabylonNative/Build_uwp_arm/**/Release/**/*.{lib,pri}')
+    .pipe(rename({ dirname: '' }))
+    .pipe(gulp.dest('Assembled/windows/libs/arm/Release'));
+}
+
+const copyARM64DebugUWPFiles = () => {
+  return gulp.src('../Modules/@babylonjs/react-native/submodules/BabylonNative/Build_uwp_arm64/**/Debug/**/*.{lib,pri}')
+    .pipe(rename({ dirname: '' }))
+    .pipe(gulp.dest('Assembled/windows/libs/arm64/Debug'));
+}
+
+const copyARM64ReleaseUWPFiles = () => {
+  return gulp.src('../Modules/@babylonjs/react-native/submodules/BabylonNative/Build_uwp_arm64/**/Release/**/*.{lib,pri}')
+    .pipe(rename({ dirname: '' }))
+    .pipe(gulp.dest('Assembled/windows/libs/arm64/Release'));
+}
+
+const copyVCXProjUWPFiles = () => {
+  return gulp.src('../Modules/@babylonjs/react-native/windows/BabylonReactNative/*.*')
+    .pipe(gulp.dest('Assembled/windows/BabylonReactNative'));
+}
+
+const copyUWPFiles = gulp.series(
```

Comment:
Could this one actually be `gulp.parallel`?

Reply:
yeah seems like it.